plugins/inventory/now.py

```diff
@@ -426,6 +431,13 @@ def add_host(self, record, name_source):
             raise AnsibleParserError(msg.format(name_source))
 
         inventory_hostname = record[name_source]
+        if isinstance(inventory_hostname, dict):
```

Comment:
You don't need this. You already set `default: false` for `display_value`. Even if the user does not explicitly use `display_value`, it will always be present in the module.